pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -16,7 +16,7 @@
 
 from .. import ModelHTTPError, UnexpectedModelBehavior, _utils, usage
 from .._output import DEFAULT_OUTPUT_TOOL_NAME, OutputObjectDefinition
-from .._utils import guard_tool_call_id as _guard_tool_call_id, number_to_datetime
+from .._utils import guard_tool_call_id as _guard_tool_call_id, number_to_datetime, now_utc as _now_utc
```

Comment:
As suggested by the linter:

```suggestion
from .._utils import guard_tool_call_id as _guard_tool_call_id, now_utc as _now_utc, number_to_datetime
```